core/src/wsv.rs

```diff
@@ -117,14 +117,9 @@ impl WorldStateView {
     #[must_use]
     #[inline]
     pub fn new(world: World) -> Self {
-        Self::from_configuration(Configuration::default(), world)
-    }
-
-    /// Add the ability of emitting events to [`WorldStateView`].
-    #[must_use]
-    pub fn with_events(mut self, events_sender: EventsSender) -> Self {
-        self.events_sender = Some(events_sender);
-        self
+        // Added to remain backward compatible with other code primary in tests
```

Comment:
Actually people normally don't want to do everything by hand, so this is a good design.